src/components/structures/auth/ForgotPassword.tsx

```diff
@@ -314,6 +322,11 @@ export default class ForgotPassword extends React.Component<IProps, IState> {
                         autoComplete="new-password"
                     />
                 </div>
+                <div className="mx_AuthBody_fieldRow">
+                    <StyledCheckbox onChange={() => this.setState({ logoutDevices: !this.state.logoutDevices })} checked={this.state.logoutDevices}>
+                        { _t("Sign out all devices") }
+                    </StyledCheckbox>
```

Comment:
We actually don't know if the server will respect this flag, it may be too old

Reply:
The UI aspect of this is resolved in the latest changes.